browser/components/MarkdownPreview.js

```diff
@@ -673,11 +674,9 @@ export default class MarkdownPreview extends React.Component {
   GetCodeThemeLink (name) {
     const theme = consts.THEMES.find(theme => theme.name === name)
 
-    if (theme) {
-      return `${appPath}/${theme.path}`
-    } else {
-      return `${appPath}/node_modules/codemirror/theme/elegant.css`
-    }
+    return theme
+      ? (win ? theme.path : `../${theme.path}`)
```

Comment:
I think this is what it needs to be for macOS: 
```suggestion
      ? (win ? theme.path : `${appPath}/${theme.path}`)
```

Reply:
OK, changed this & tested it with Ubuntu binary. Theming (Editor, in preview code block & app theme) is working.